apps/emqx_management/test/emqx_mgmt_api_clients_SUITE.erl

```diff
@@ -1072,6 +1080,7 @@ get_clients(Auth, Qs, ExpectError, ClientIdOnly) ->
     Resp = emqx_mgmt_api_test_util:request_api(get, ClientsPath, Qs, Auth),
     case ExpectError of
         false ->
+            ct:pal("get clients response:\n  ~p", [Resp]),
```

Comment:
Nit. This looks redundant if the test code matches on important parts of a response anyway?

Reply:
True, I think I was debugging and forgot this print.  :see_no_evil: